src/features/auth/components/AuthForm/AuthForm.css

```diff
@@ -50,26 +49,37 @@
 }
 
 .auth-form__button {
+  position: relative;
   margin: 0 auto;
+  outline: none;
```

Comment:
it works in Chrome on Windows?
i remember it only worked for me to reset outline on :focus